kolibri/content/test/test_content_app.py

```diff
@@ -315,7 +315,7 @@ def test_channelmetadata_content_available_param_filter_lowercase_true(self):
 
     def test_channelmetadata_content_available_param_filter_uppercase_true(self):
         response = self.client.get(reverse("channel-list"), {"available": True})
-        self.assertEqual(response.data, [])
+        self.assertEqual(response.data[0]["id"], "6199dde695db4ee4ab392222d5af1e5c")
```

Comment:
Is this expected?

Reply:
Yes, thanks to the new django filter boolean filter in drf which is more permissive, and evaluates a range of truthy things!